Security/centurylink-cloud-guide-to-vormetric-dsm.md

```diff
@@ -491,55 +491,55 @@ DSM (Primary)|DSM (Failover)|One Way|TCP/50000
+[Standard Virtual Server](../Servers/centurylink-cloud-backup-and-recovery-services.md)|Standard Virtual Servers include either a 5 or 14 day daily virtual server backup of the entire instance. Using this service with encrypted data sets should be transparent during a backup or virtual server restore action as the entire virtual server is impacted. Host agents will communicate with the Vormetric DSM and receive the same policy with key access even if you restore a virtual server to a previous state.
+[Simple Backup Service](//www.ctl.io/simple-backup-service)|The Simple Backup Service leverages a client agent to backup and restore data to the server. As part of policy creation a security rule must be put in place that **permits** the backup agent executable **read/write** to the GuardPoint. Agent based backups are the only situation in which a security rule should permit read/write to a GuardPoint. It's important to note that should you restore data using the agent to a location outside of the GuardPoint policy configuration, changes will be required in order to permit access to this new location and apply the encryption key. An example policy that includes a security rule for a backup agent can be seen below.
+[Managed Backup](//www.ctl.io/managed-services/backup)|Managed Backup services leverage client agents similar to Simple Backup Services. As such, the rules and approach would mirror the process already detailed.
+[Ecosystem](../Ecosystem Partners/General/ecosystem-partner-list.md)|3rd Party Ecosystem partners such as [Commvault](../Ecosystem Partners/Marketplace Guides/getting-started-with-commvault-storage-blueprint.md) generally leverage client agents similar to Managed Backup services. As such, the rules and approach would mirror the process already detailed.
+[Snapshots](../Servers/creating-and-managing-server-snapshots.md)|While a snapshot is not technically a backup, it does facilitate the restoration of an entire virtual server to a previous state. Using this service with encrypted data sets should be transparent during a snapshot or revert snapshot action as the entire virtual server is impacted. Host agents will communicate with the Vormetric DSM and receive the same policy with key access even if you revert a snapshot to a previous state.
 
 **Example Unstructured Data Policy**
 
 **Policy**|**Resource**|**User**|**Process**|**Action**|**Effects**
 ----------|------------|--------|-----------|----------|-----------
-**1**|&#42;|Trust_Set|&#42;|&#42;|Permit<br>Apply Key
-**2**|&#42;|&#42;|Backup_Set|Read/Write|Permit
-**3**|&#42;|&#42;|&#42;|&#42;|Deny<br>Audit
+**1** |&#42;|Trust_Set|&#42;|&#42;|Permit<br>Apply Key
+**2** |&#42;|&#42;|Backup_Set|Read/Write|Permit
+**3** |&#42;|&#42;|&#42;|&#42;|Deny<br>Audit
```

Comment:
there are no need for spaces in this table code, or anywhere else.  it conforms to markdown standards and renders